Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -53,6 +54,8 @@ enum FramesLogEvent: Equatable, PropertyProviding {
             return "payment_form_initialised"
         case .paymentFormPresented:
             return "payment_form_presented"
+        case .paymentFormSubmitted:
+            return "payment_form_submitted"
```

Comment:
@maxim-nosov-cko @chintan-soni-cko @patrick-hoban-cko will have to create similar events for Android as well in the backlog once this PR is approved and merged.